src/resmom/requests.c

```diff
@@ -1559,14 +1559,7 @@ post_suspend(job *pjob, int err)
 		return;
 
 	if ((pjob->ji_flags & MOM_SISTER_ERR) == 0) {
-		/* save stop time for adjusting walltime */
-		pjob->ji_momstat = time_now;
```

Comment:
So it's looking like job's ji_momstat is no longer used. Perhaps this attribute can be removed from the job structure.